receiver/k8sclusterreceiver/mock_exporter.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020, OpenTelemetry Authors
```

Comment:
If this is only need for tests might be better to end it in _test so it only gets built then.